lib/astarte/client/appengine/devices.ex

```diff
@@ -44,7 +77,20 @@ defmodule Astarte.Client.AppEngine.Devices do
     end
   end
 
-  def get_device_interfaces(%AppEngine{} = client, device_id) do
+  @doc """
+  Returns a list of interfaces supported by a certain device.
+
+  Interfaces that are not reported by the device are not reported here.
+  If a device stops to advertise a certain interface, it should be retrived from a different API,
```

Comment:
If a device stops to advertise a certain interface, it should be retrived from a different API **.**